### DISCUSSION TITLE
Duda HTML Logo Mensaje

### DISCUSSION THREAD

**Discussion by MiguelARojas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#208] [Q&A] [answered]
Que tal tengo una duda, en la parte donde dice: Crear marca de la pastelería como un logo y los mensajes que daremos desde la marca.
¿ A que se refiere exactamente con la parte del mensaje?

**Comment by AngelCruzO** [comment, marked as answer]
Eslogan del negocio, mensajes para marketing

**Comment by JuanSalvador73** [comment]
A mi la verdad me suena mas como "slogan", incluso texto de información de que es lo que ofrece y hace la pastelería. Porque mensaje como tal, solo sería ya que tuviésemos un formulario completo y funcional, y no hemos llegado a esa etapa.

**Comment by hackit92** [comment]
Complementando lo que dice AngelCruz, desde la perspectiva del MKT el sitio debe tener una línea de comunicación que decir, es decir... si nuestra marca será una pastelería gourment todo girará en torno a lo que el publico objetivo quiere escuchar de una marca así, pro ende deberíamos evitar decir que somos "los más baratos de la región" y sería mejor incluir algo como "Los mejores pasteles del estado"
